src/reducers/user.reducer.js

```diff
@@ -1,21 +1,16 @@
 import { userConstants } from '../constants/user.constants';
 
-export const authentication = (state = {}, { type, result }) => {
+export const user = (state = {}, { type, result }) => {
   switch (type) {
     case userConstants.USER_REQUEST:
       return {
         userRequest: true,
```

Comment:
What about `loading`? Or `ongoingUserRequest` if you want to maintain the user request semantics... But I think it sbhould be something that tells me that there is something ongoing/pending.